client/src/pages/automation/project/components/node-details-tabs/OutputTab.tsx

```diff
@@ -205,25 +245,34 @@ const OutputTab = ({
                     </div>
 
                     <div className="mt-2 flex items-center">
-                        <div className="flex items-center">
+                        <div className="group relative flex items-center rounded-md p-1 pr-5 hover:bg-gray-100">
```

Comment:
jel triba dupli div? mislin da se mogu kombinirati u jedan

Reply:
ne diram sada, to ćeš ti